packages/api/src/record.ts

```diff
@@ -100,13 +103,11 @@ export class IPNSRecord {
       return jsonResponse(JSON.stringify(data), 200)
```

Comment:
We could set the HTTP code to 201 when the record is created here.

Reply:
Great suggestion. Done.